src/ProjectService.Models.Dto/Requests/Filters/FindProjectsFilter.cs

```diff
@@ -1,8 +1,18 @@
 ﻿using LT.DigitalOffice.Kernel.Requests;
+using LT.DigitalOffice.ProjectService.Models.Dto.Enums;
+using Microsoft.AspNetCore.Mvc;
 
 namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests.Filters
 {
   public record FindProjectsFilter : BaseFindFilter
   {
+    [FromQuery(Name = "isascendingsort")]
+    public bool? IsAscendingSort { get; set; }
+
+    [FromQuery(Name ="projectstatus")]
```

Comment:
you in project filter used by project controller, I think there no need write the word "project", may be better "hasStatus"?